index.md

```diff
@@ -2,8 +2,52 @@
 site: sandpaper::sandpaper_site
 ---
 
-This is a new lesson built with [The Carpentries Workbench][workbench]. 
+<img src="https://github.com/carpentries-incubator/scientific-metadata/blob/awesome_dev/episodes/fig/Welcome.png?raw=true/" alt="Welcome." height="75" align="left">
+<br><br><br><br>
+
+to our entry-level introduction to the **fundamentals of scientific metadata**!
+
+In this lesson we will look at the intricate relationship between (digital) research data, metadata and knowledge, discuss why metadata is critical in today's research, as well as explain some of the technologies and concepts related to structured machine-readable metadata.
+
+Have you ever struggled to make sense of scientific data provided by a collaborator - or even understanding your own data 5 months after publication? Do you see difficulties in meeting the data description requirements of your funding agency? Do you want your data to have lasting value, but don’t know how to ensure that?
+
+Precise and structured description of research data is key for scientific exchange and progress - and also for the recognition of your effort in data collection. The solution: make your data findable, accessible, interoperable and reusable by describing them with metadata.
+
+This course is targeted at scientific staff and researchers from all fields, who would like to better understand research data annotation with metadata.<br><br>
+
+<img src="https://github.com/carpentries-incubator/scientific-metadata/blob/awesome_dev/episodes/fig/youWillLearn.png?raw=true"/ alt="Content overview. In this lesson you will learn, of the differences between and the importance of data and metadata, how to annotate your research data with structured metadata, how to find and evaluate suitable metadata frameworks, how to use basic Markdown, JSON, and JSON Schema, about available tools that you can use to up your metadata annotation game, why structured metadata is important and how it can increase your scientific visibility.">
+<br>
 
+**This lesson is for you, if...**
+<ul style="list-style:none">
+  <li>... you are completely new to the topic of scientific metadata.</li>
+  <li>... you are looking for alternatives for documenting your research data.</li>
+  <li>... you have a basic understanding of metadata but want to gain some hands-on experience.</li>
+  <li>... you want to get into the function and syntax of JSON and JSON schema.</li>
+  <li>... you are looking for metadata standards and tools.</li>
+</ul><br>
 
-[workbench]: https://carpentries.github.io/sandpaper-docs
+**You might want to look for advanced lessons, if...**
```

Comment:
Are there concrete advance lessons you would recommend? The sentence "You might want to look for advanced lessons" may suggest that advance lessons are also available. If this is not the case maybe use something like "This lesson may not be what you are looking for if..." instead?